src/components/layout/playground/requests/requestCard.tsx

```diff
@@ -132,17 +131,20 @@ const PlaygroundRequestCard: React.FC<
           >
             {requester.name}
           </Li>
-          <Li
-            title={`${timeUntilDue ? (isDue ? 'Was due' : 'Due in') : 'Due'} ${
-              timeUntilDue || ''
-            }
-            ${timeUntilDue ? (isDue ? ' ago' : '') : 'Today'}`}
-            name={`${timeUntilDue ? (isDue ? 'Was due' : 'Due in') : 'Due'}`}
-            category={category}
-          >
-            {timeUntilDue}
-            {timeUntilDue ? (isDue ? ' ago' : '') : 'Today'}
-          </Li>
+          {hasNoDue ? (
+            <Li title={'Due Date'} name={'Due Date'} category={category}>
+              None
+            </Li>
+          ) : (
+            <Li
+              title={`${timeUntilDue ? (isDue ? 'Was due' : 'Due in') : 'Due'}`}
+              name={`${timeUntilDue ? (isDue ? 'Was due' : 'Due in') : 'Due'}`}
+              category={category}
+            >
+              {timeUntilDue}
+              {timeUntilDue ? (isDue ? ' ago' : '') : 'Today'}
+            </Li>
+          )}
```

Comment:
Same goes for this one:
```suggestion
            <Li
              title={`${hasNoDue ? 'Due Date' : timeUntilDue ? (isDue ? 'Was due' : 'Due in') : 'Due'}`}
              name={`${hasNoDue ? 'Due Date' : timeUntilDue ? (isDue ? 'Was due' : 'Due in') : 'Due'}`}
              category={category}
            >
              {hasNoDue ? 'None' : timeUntilDue}
              {!hasNoDue ? (timeUntilDue ? (isDue ? ' ago' : '') : 'Today') : ''}
            </Li>
```